### PR TITLE
Fixes code examples in README

### PR DESCRIPTION
**Pull request by skaterdav85**

### CODE REVIEW

**Review by toh995 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ After:
 ```
 
 ```js
-export default class Component.extend({
+export default Component.extend({
```
Comment (rwjblue) [inline README.md:74]: Hmm, let's go the other direction (if you don't mind); Let's do this:

```js
export default class extends Component {
```

What do you think @skaterdav85?

Comment (skaterdav85) [reply README.md:74]: @rwjblue sounds good. I'll update.

Comment (skaterdav85) [reply README.md]: @rwjblue done

File: README.md [hunk 2]
```diff
@@ -166,18 +166,18 @@ Node component:
...
+  constructor() {
     super(...arguments);
     this.children = new Set();
 
-    this.parent.registerChild(this);
+    this.args.parent.registerChild(this);
   }
 
   willDestroy() {
     super(...arguments);
```
Comment (rwjblue) [inline README.md:178]:
```suggestion
    super.willDestroy(...arguments);
```

**Pull request merged**